android/app/src/main/java/com/reactnativenavigation/params/parsers/StyleParamsParser.java

```diff
@@ -125,6 +127,8 @@ private StyleParams createDefaultStyleParams() {
         result.titleBarHideOnScroll = false;
         result.orientation = Orientation.auto;
         result.bottomTabFontFamily = new StyleParams.Font();
+        result.bottomTabFontSize = 10;
```

Comment:
We should avoid hardcoding a default font size. Can we set font size only if specified by the user?